ui/apps/platform/src/Containers/Policies/Table/PoliciesTablePage.tsx

```diff
@@ -88,7 +88,10 @@ function PoliciesTablePage({
             });
     }
 
-    function fetchPolicies(query: string, sortOption: ApiSortOption) {
+    function fetchPolicies(query: string, fetchSortOption: ApiSortOption) {
+        // The policy table does not currently support multi sort, but it must handle the case where the sortOption is an array
+        // due to the hook's API. Although this should not occur, we will handle it here by using the first option.
+        const sortOption = Array.isArray(fetchSortOption) ? fetchSortOption[0] : fetchSortOption;
```

Comment:
This is the only location in the app where the `useURLSort()` hook is used for _client side_ pagination. Since the standard helper functions for server side pagination handle the array case internally, we only need to explicitly handle it here.

Currently the array case will never occur given the implementation of this table, but since it is technically possible we handle it by using the first sort option.